source/_patterns/30-organisms/collection/collection.scss

```diff
@@ -5,60 +5,77 @@
 .collection {
   display: flex;
   flex-direction: column;
-  margin: 8px 8px 16px;
+  margin-top: 8px;
+  margin-right: 8px;
+  margin-bottom: 16px;
+  margin-left: 8px;
   border-radius: 2px;
   background-color: #f5f5f2;
-  box-shadow: 0 0 1px 0 rgba(0, 0, 0, 0.24), 0 0 1px 0 rgba(0, 0, 0, 0.12);
+  box-shadow: $shadow-elevation1;
 }
 
 .collection__title {
-  margin: 16px 24px 12px;
+  margin-top: 16px;
+  margin-right: 24px;
+  margin-bottom: 12px;
+  margin-left: 24px;
   font-family: $font-medium;
   font-size: 24px;
   font-weight: 500;
   font-style: normal;
   font-stretch: normal;
   line-height: 1.25;
-  letter-spacing: 0.2px;
-  color: #22211d;
+  letter-spacing: .2px;
+  color: $color-text-default;
 
   @include tablet-up {
-    margin: 16px 32px 12px;
+    margin-right: 32px;
+    margin-left: 32px;
   }
 
   @include desktop-up {
-    margin: 20px 32px 16px;
+    margin-top: 20px;
     font-size: 28px;
   }
 
   @include desktop-wide {
-    margin: 20px 40px 16px;
+    margin-right: 40px;
```

Comment:
ich hab extra alle Werte für die versch. BPs notiert, jetzt muss man schauen was der Standard-Wert ist und ob er in einem anderen BP bereits überschrieben wird. Man muss wissen dass desktop-wide grösser als desktop ist und dass desktop-up auch für desktop-wide gilt. Ich bin sicher dass hier schnell was vergessen geht und würde darum wieder zurück zur vollständigen Auflistung gehen